docs/guides/creatingasplitter.md

```diff
@@ -1,17 +1,27 @@
-# Creating a Splitter
+# Creating an Event Splitter
```

Comment:
Same question about the page title. Not sure when it's needed and when it's not, so this is just a reminder to double check.